ddtrace.gemspec

```diff
@@ -33,6 +33,7 @@ EOS
   spec.require_paths = ['lib']
 
   spec.add_dependency 'msgpack'
+  spec.add_dependency 'concurrent-ruby'
```

Comment:
doesn't this introduce a strong dependency in our tracer? we should keep our dependency list at minimum, that in our case is just `msgpack`. Am I missing something?

Reply:
Yes, but only at `bundle` or `gem install` time. It doesn't get loaded unless you are using the `TestTracer`. I'm fine skipping it. That gem has nice concurrency primitives. It is much nicer to use the manually toggling mutexes. If the `FauxTransport` and `SpyTransport` get refactored out of the `test/helper.rb` and into a public API, I'll just drop this PR and use those instead.